support-services/src/main/scala/com/gu/support/redemption/WithLogging.scala

```diff
@@ -0,0 +1,31 @@
+package com.gu.support.redemption
+
+import com.gu.monitoring.SafeLogger
+
+import scala.concurrent.{ExecutionContext, Future}
+
+trait WithLogging {
+
+  implicit class LogImplicit2[A](value: Future[A]) {
+
+    // this is just a handy method to add logging to the end of any value
+    def withLoggingAsync(message: String)(implicit executionContext: ExecutionContext): Future[A] = {
```

Comment:
This already exists here: https://github.com/guardian/support-frontend/blob/e6b51f248825528661f5160b5dc86498284a4598/support-lambdas/stripe-intent/src/main/scala/com/gu/handler/LogImplicit.scala#L12

Reply:
yes I didn't think it was worth adding a dependency just for a bit of syntactic sugar so I just copied and pasted it.  There are a couple of copies already in support-frontend repo.  Happy to consider alternative approaches though.